src/views/AmalgamationSelection.vue

```diff
@@ -225,45 +223,38 @@ export default class AmalgamationSelection extends Vue {
    */
   private async createBusinessAA (type: AmalgamationTypes): Promise<string> {
     const accountId = +JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id || 0
-    const legalType = this.getLegalType
-    const email = type === AmalgamationTypes.HORIZONTAL ||
-      type === AmalgamationTypes.VERTICAL ? this.getBusinessEmail : ''
-    const phone = type === AmalgamationTypes.HORIZONTAL ||
-      type === AmalgamationTypes.VERTICAL ? this.getFullPhoneNumber : ''
-    const correctNameOption =
-      type === AmalgamationTypes.HORIZONTAL ||
-      type === AmalgamationTypes.VERTICAL
-        ? CorrectNameOptions.CORRECT_AML_ADOPT
-        : null
-    const legalName =
-      correctNameOption === CorrectNameOptions.CORRECT_AML_ADOPT
-        ? this.getLegalName
-        : null
+    const email = this.isShortFormAmalgamation(type) ? this.getBusinessEmail : ''
+    const phone = this.isShortFormAmalgamation(type) ? this.getFullPhoneNumber : ''
+    const legalName = this.isShortFormAmalgamation(type) ? this.getLegalName : ''
+    const correctNameOption = this.isShortFormAmalgamation(type)
+      ? CorrectNameOptions.CORRECT_AML_ADOPT
+      : CorrectNameOptions.CORRECT_AML_NUMBERED
```

Comment:
This is the change for regular amalgamations. (And there is no legal name for the numbered option.)